params.go

```diff
@@ -21,6 +21,8 @@ import (
 	"regexp"
 	"sort"
 	"strings"
+
+	"github.com/deckarep/golang-set"
```

Comment:
Aside: I see that this set has a contains method. Maybe use this more when array membership requires a loop :)